beta/vercel.json

```diff
@@ -254,5 +254,16 @@
       "destination": "https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com",
       "permanent": false
     }
+  ],
+  "headers": [
+    {
+      "source": "/fonts(.*)",
```

Comment:
Shall we do something like `/fonts/(.*).woff2`? instead of opening up entire fonts ?